iOS-Email-Client/Libs/Env.swift

```diff
@@ -63,21 +63,21 @@ struct Env {
         guard !Env.isProduction else {
             return "criptext.com"
         }
-        return "criptext.com"
+        return "jigl.com"
     }
     
     static var apiURL: String {
         guard !Env.isProduction else {
             return "https://api.criptext.com"
         }
-        return "https://api.criptext.com"
+        return "https://stage.mail.criptext.com"
     }
     
     static var transferURL: String {
         guard !Env.isProduction else {
             return "https://transfer.criptext.com"
         }
-        return "https://transfer.criptext.com"
+        return "https://stagetransfer.criptext.com"
```

Comment:
PR should not have staging urls
```suggestion
        return "https://transfer.criptext.com"
```